audio/tests/audiotools/data/test_transforms.py

```diff
@@ -0,0 +1,457 @@
+# MIT License, Copyright (c) 2023-Present, Descript.
+# Copyright (c) 2025 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Modified from audiotools(https://github.com/descriptinc/audiotools/blob/master/tests/data/test_transforms.py)
+import inspect
+import sys
+import warnings
+from pathlib import Path
+
+import numpy as np
+import paddle
+import pytest
+
+sys.path.append("../..")
+import audiotools
+from audiotools import AudioSignal
+from audiotools import util
+from audiotools.data import transforms as tfm
+from audiotools.data.datasets import AudioDataset
+from paddlespeech.vector.training.seeding import seed_everything
+
+non_deterministic_transforms = ["TimeNoise", "FrequencyNoise"]
+transforms_to_test = []
+for x in dir(tfm):
+    if hasattr(getattr(tfm, x), "transform"):
+        if x not in ["Compose", "Choose", "Repeat", "RepeatUpTo"]:
+            transforms_to_test.append(x)
+
+
+def _compare_transform(transform_name, signal):
+    regression_data = Path(f"regression/transforms/{transform_name}.wav")
+    regression_data.parent.mkdir(exist_ok=True, parents=True)
+
+    if regression_data.exists():
+        regression_signal = AudioSignal(regression_data)
+        try:
+            assert paddle.allclose(
```

Comment:
can  precision pass 1e-6?

Reply:
In original code, the number is also `1e-4`
https://github.com/descriptinc/audiotools/blob/348ebf2034ce24e2a91a553e3171cb00c0c71678/tests/data/test_transforms.py#L28